modules/acquia_cms_common/src/Services/ConfigImporterService.php

```diff
@@ -209,7 +210,7 @@ public function doImport(StorageComparer $storage_comparer, DrushLoggerManager $
         $message = 'The import failed due to the following reasons:' . "\n";
         $message .= implode("\n", $config_importer->getErrors());
 
-        watchdog_exception('config_import', $e);
+        Error::logException('config_import', $e);
```

Comment:
@rajeshreeputra prior to D10.1 this would throw an undefined method error ~~ can we add version_compare logic?
cc: @vishalkhode1

Reply:
added version compare